README.md

```diff
@@ -1,134 +1,32 @@
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Platform Node.js Samples [![Slack][slack_badge]][slack_link] [![Coverage][cov_badge]][cov_link]
```

Comment:
This kind of makes me sad :(  Are we making a choice not to hang in the slack?

Reply:
I'm never in Slack. Is anybody?

Reply:
Adding the link to Slack back in